Offline/OfflinePlayer/objc/OfflinePlayer/Managers and Utilities/DownloadManager.m

```diff
@@ -241,8 +241,22 @@ - (void)downloadVideoFromQueue
      
      }];
     
+    // Prior to iOS 13 it was possible to download secondary tracks separately from the video itself.
+    // On iOS 13+ you must now download secondary tracks along with the video.
+    // The existing method for downloading videos is: `requestVideoDownload:parameters:completion:`
+    // You may still use this method on iOS 11 and 12.
+    // If you want to support iOS 13 and do not want to have any branching logic
+    // you can use the new method that is backwards compatible:
+    // `requestVideoDownload:mediaSelections:parameters:completion:`
+    
     __weak typeof(self) weakSelf = self;
+    AVURLAsset *avURLAsset = [self.offlineVideoManager urlAssetForVideo:video error:nil];
+    NSArray<AVMediaSelection *> *mediaSelections = @[avURLAsset.preferredMediaSelection];
+    if (@available(iOS 11.0, *)) {
+        mediaSelections = [avURLAsset allMediaSelections];
```

Comment:
This part confuses me. Why choose the `preferredMediaSelection` and then replace it with `allMediaSelections`? And how come one uses the dot operator and the other uses a method call? For my 2 cents, give the user one example using `allMediaSelections`. I suggest `allMediaSelections` because preferredMediaSelection is the default value when the caller passes `nil` for `mediaSelections`.

Reply:
I defaulted to `preferredMediaSelection` here because `allMediaSelections` is only available on iOS 11+. I considered passing just nil since that was the case but figured I'd be explicit so they can clearly see what will be downloaded in the case they don't read the documentation.

It does make sense to use dot operator for `allMediaSelections` as well, not sure why I did that.